cmd/set.go

```diff
@@ -52,3 +74,8 @@ func SetCommand() cli.Command {
 		},
 	}
 }
+
+func SetProxyPort(port int) (p string) {
```

Comment:
`SetProxyPort` is not a good name for this function. It is not "setting" anything, rather just constructing a proxy URL. `MakeProxyURL` would be a better name, or something else...

Reply:
Done